pwnlib/config.py

```diff
@@ -55,8 +55,8 @@ def initialize():
     from pwnlib.log import getLogger
     log = getLogger(__name__)
 
-    xdg_config_home = (os.environ.get('XDG_CONFIG_HOME') or
-                       os.path.expanduser("~/.config"))
+    xdg_config_home = os.environ.get('XDG_CONFIG_HOME',
```

Comment:
Does this work? `os.path.join((a,b), c)`?

Reply:
No that would return:
```python
TypeError: expected str, bytes or os.PathLike object, not tuple
```
What is your intention for `os.path.join((a,b), c)`? Is perhaps `os.path.join(a, b, c)` what you are looking for?

Reply:
And this is just the error that you would get running this code as far as I see. Or I am wrong, which may be the case as well.

Reply:
@Arusekk I am bit confused on what you are asking or inferring.
The original syntax is:
```python
xdg_config_home = (os.environ.get('XDG_CONFIG_HOME') or os.path.expanduser("~/.config"))
```
I have changed it to:
```python
xdg_config_home = os.environ.get('XDG_CONFIG_HOME', os.path.expanduser("~/.config"))
```
which maintains the original behavior but with a more Pythonic syntax. `xdg_config_home` in both cases is either the value of `os.environ.get('XDG_CONFIG_HOME') ` or `os.path.expanduser("~/.config")` depending on if `os.environ.get('XDG_CONFIG_HOME')` returns a `True` or equivalent of a `True` value.

Are you referring to line 62-64:
```python
c.read(['/etc/pwn.conf',
        os.path.join(xdg_config_home, 'pwn.conf'),
        os.path.expanduser('~/.pwn.conf')])
```
I have left code snippet unchanged. And I only see a function call on a list in the form of:
```python
[filepath_string,
os.path.join(a, b),
os.path.expanduser(another_filepath_string)]
```
I don't see where `os.path.join((a,b), c)` is coming from with the changes I have made.

Reply:
Ah sure, so sorry for wasting your time on that.